static/state_bydate.js

```diff
@@ -396,6 +394,47 @@ function parsingDone(results, file) {
   });
 }
 
+var groupToEssentialMap = new Map();
+
+var groupMappings = [
+  {groupName:"Airport",essential:true},
+  {groupName:"Alcohol",essential:true},
+  {groupName:"Arts & Entertainment",essential:false},
+  {groupName:"Banks",essential:true},
+  {groupName:"Beach",essential:false},
+  {groupName:"Big Box Stores",essential:false},
+  {groupName:"Bus",essential:true},
+  {groupName:"Colleges & Universities",essential:false},
+  {groupName:"Convenience Store",essential:true},
+  {groupName:"Discount Stores",essential:false},
+  {groupName:"Drug Store",essential:true},
+  {groupName:"Fast Food Restaurants",essential:true},
```

Comment:
in a pandemic nobody should eat fast food. staying healthy is hard enough as it is